backend/server.js

```diff
@@ -0,0 +1,96 @@
+import session from 'express-session';
+import cors from 'cors';
+import morgan from 'morgan';
+// import { User, Post } from './models/index.js';
+import userRoutes from './routes/users.js';
+// const {PrismaClient} = require('@prisma/client');
+import {PrismaClient} from '@prisma/client'
+const prisma = new PrismaClient();
+
+import express from 'express'
+const app = express();
+
+// require('dotenv').config()
+import {} from 'dotenv/config'
+const PORT = process.env.PORT || 3000;
+
+app.use(cors({
+  origin: `http://localhost:${PORT}`,
+  credentials: true
+}));
+app.use(express.json()); // Middleware for parsing JSON bodies from HTTP requests
+app.use(morgan())
+
+app.listen(PORT, () =>
+  console.log(`Server running on http://localhost:${PORT}`
+));
+
+// Session middleware
+app.use(
+  session({
+    secret: 'your-secret-key',
+    resave: false,
+    saveUninitialized: false,
+    cookie: {
+      sameSite: false,
+      secure: false,
+      expires: new Date(Date.now() + (365 * 24 * 60 * 60 * 1000)) // 1 year in milliseconds
+    }
+  })
+);
+
+app.use(userRoutes);
+
```

Comment:
Ignore the entire rest of this file, it's all commented out code. I get it now - I need to clean up my code before commits too, not just for pushes. Silly me.